.github/workflows/publish_snapshot.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         java-version: 8
 
     - name: Release Maven package
-      uses: samuelmeuli/action-maven-publish@v1
+      uses: samuelmeuli/action-maven-publish@v1.4.0
```

Comment:
According to the error message, I'm not sure this will work here. Do you have any reference why this fix the current error?
```
samuelmeuli/action-maven-publish@v1 is not allowed to be used in apache/orc.
Actions in this workflow must be: created by GitHub, verified in the GitHub Marketplace,
within a repository owned by apache or match the following:
adoptopenjdk/*, apache/*, gradle/wrapper-validation-action.
```

Reply:
> Are you sure that your claim snapshot publishing is broken by ORC-704?
You are right, 704 is not related -- just happened that all commits after that failed to publish a snapshot.

Regarding the error message: the confusion comes from the fact that the repo is actually in the apache/* form (thus assumed it was a string/expression evaluation issue). 

>  Do you have any reference why this fix the current error?

No ref here, just wanted to check if this helps with the policy issue above (from the message I assume we shouldn't be seeing the issue and it could be a bug).
Please let me know If there are other things we could try to fix this, you definitely have more experience GH actions :)

Reply:
Ya, +1 for that.